scipy/sparse/linalg/tests/test_expm_multiply.py

```diff
@@ -82,6 +82,7 @@ def test_onenormest_matrix_power(self):
                 assert_(less_than_or_close(estimated, exact))
                 assert_(less_than_or_close(exact, 3*estimated))
 
+    @pytest.mark.thread_unsafe
```

Comment:
Reverted. I've moved all changes to this module to https://github.com/scipy/scipy/pull/23305